test/00_operators/OperatorTests.cu

```diff
@@ -1846,6 +1846,33 @@ TYPED_TEST(OperatorTestsNumericAllExecs, OperatorFuncs)
   res = c * c * (c + c) / c + three;
   EXPECT_TRUE(MatXUtils::MatXTypeCompare(tov0(), res, 0.07));
 
+  // example-begin nan-test-1
+  auto nan = make_tensor<TestType>({});
+  using conversionType = typename matx::detail::value_promote_t<TestType>;  
+  if constexpr(matx::is_complex_v<TestType>) {    
+    nan() = TestType(std::numeric_limits<conversionType>::quiet_NaN());
+  } else {
+    nan() = std::numeric_limits<conversionType>::quiet_NaN();
+  }
+  auto tob = make_tensor<bool>({});
+  (tob = matx::isnan(nan)).run(); 
+  cudaDeviceSynchronize();  
+  EXPECT_TRUE(MatXUtils::MatXTypeCompare(tob(), std::is_floating_point_v<conversionType> ? true : false));
+  // example-end nan-test-1
```

Comment:
The example section should only include the use of the operator and possibly some comments explaining it. The rest of the stuff is just for boiler plate unit test stuff.